Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs

```diff
@@ -25,11 +25,11 @@ public void Construct(IUIRoot uiRoot, LoadingService loadingService, LevelLoadin
 
         protected void Start()
         {
-            //var view = _uiRoot.Open<StartSplashScreen>();
-            //_uiRoot.Open<MainMenu>();
-            //view.Closed += OnViewClosed;
-            var view = _uiRoot.Open<StartSplashScreen>();
-            view.Closed += OnSplashScreenClosed;
+            var mainMenuView = _uiRoot.Open<MainMenu>();
```

Comment:
This Behaviour must be invoce after start splash screen closed/ Pls move to anpther method

Reply:
Done